crates/freeze/src/datasets/erc20_transfers.rs

```diff
@@ -37,7 +37,28 @@ impl CollectByBlock for Erc20Transfers {
         let topics = [Some(ValueOrArray::Value(Some(*EVENT_ERC20_TRANSFER))), None, None, None];
```

Comment:
from_address and to_address are indexed for erc20 and erc721 addresses in topic1 and topic2. so they can go as items in this filter. it will fetch less data and be simpler than filtering

Reply:
resolved in [55aa899](https://github.com/paradigmxyz/cryo/pull/123/commits/55aa899236c61f878b8159c8e6c42da29c8478c7)